pkg/qdr/qdr.go

```diff
@@ -741,8 +741,9 @@ type TcpEndpointDifference struct {
 }
 
 type HttpEndpointDifference struct {
-	Deleted []HttpEndpoint
-	Added   []HttpEndpoint
+	Deleted          []HttpEndpoint
+	Added            []HttpEndpoint
+	AddedSslProfiles []string
```

Comment:
Would this field make more sense on the BridgeConfigDifference?

Reply:
Yes indeed, sslProfiles are independent entities. I will change that. Thanks!